### DISCUSSION TITLE
Mesh not starting automatically

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5901] [Issues]
Hi all,

I have a a service which runs in the background and everything was working find.
I entered ServerUpdate command in the console of the server and since then the site is not available although the service still runs.
I am able to run it manually but something changed after the ServerUpdate and I'm not sure what caused it.
I ran the 1.21 version and tried to update back to the stable version but it didn't help.
Any idea would be appriciated

**Comment by si458** [comment]
How did u install it originally? Windows installer? Npm install? 
Windows server or Linux?
Have u tried turning the server off and on again?

**Comment by ghost** [comment]
Silly me, it's a Linux Mint installed by NPM
I just saw that locally it works but I need to run the node command manually to connent via my domain.
I tried running the sudo setcao command with which node but it still gives the error that the Mesh is running without permissions below 1025

**Comment by ghost** [comment]
OK it works again and I'm not sure what solved it but I think it's because I went back to the stable 1.1.0 version
